wallet/test/de/schildbach/wallet/util/viewModels/MainViewModelTest.kt

```diff
@@ -135,6 +138,10 @@ class MainViewModelTest {
         every { observe(WalletUIConfig.SELECTED_CURRENCY) } returns MutableStateFlow("USD")
     }
 
+    private val userAgentDaoMock = mockk<UserAlertDao> {
+        every { observe(any()) } returns flow { }
+    }
```

Comment:
I'm not sure why this problem started happening now and not previously on the dashpay branch.  

`observeBlockchainState_replaying_notSynced` started to fail.